bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Row, Col, ColAuto, ColAutoRight } from './Grid';
+import { Slider, Icon, NumberInput } from 'oskari-ui';
+import { EyeOpen, EyeShut, DataLayerIcon, ImageLayerIcon, DragIcon } from './CustomIcons/CustomIcons';
+
+const StyledBox = styled.div`
+    min-height: 100px;
+    box-shadow: 1px 1px 3px 0 rgba(0, 0, 0, 0.23);
+    border-radius: 3px;
+    margin-bottom: 10px;
+    margin-left: 5px;
+    margin-right: 5px;
+    border: 1px #fff solid;
+`;
+
+const GrayRow = styled(Row)`
+    background-color: #f3f3f3;
+    padding-left: 60px;
+`;
+
+const StyledSlider = styled.div`
+    border: solid 2px #d9d9d9;
+    border-radius: 4px;
+    width: 150px;
+    padding: 8px 15px;
+`;
+
+const StyledNumberInput = styled(NumberInput)`
+    width: 70px !important;
+    font-size: 15px;
+    box-shadow: inset 1px 1px 4px 0 rgba(87, 87, 87, 0.26);
+`;
+
+const LayerIcon = ({ type }) => {
+    if (type === 'wmts') {
```

Comment:
wms also uses ImageLayerIcon and there's no handling for userdata/timeseries etc. Add a TODO or add missing types.

Reply:
This will be handled on the next ticket.